tests/unit_tests/ensemble_evaluator/ensemble_evaluator_utils.py

```diff
@@ -103,13 +103,6 @@ def _evaluate(self, url):
             event_id = event_id + 1
             for real in range(0, self.test_reals):
                 job_failed = False
-                send_dispatch_event(
```

Comment:
Should we send a corresponding `REALIZATION_UNKNOW` status event?

Reply:
Added that. But don't know what it is good for.